src/docs/product/relay/options.mdx

```diff
@@ -6,34 +6,42 @@ redirect_from:
   - /meta/relay/options/
 ---
 
-Configuration for Relay is recorded in the file `.relay/config.yml`. To change this location, pass the `--config` option to any Relay command:
+Configuration for Relay is recorded either in the file `.relay/config.yml`, or when noted, can be overridden through an environment variable. To change the config location, pass the `--config` option to any Relay command:
 
 ```shell
 ❯ ./relay run --config /path/to/folder
 ```
 
 All configuration keys are `snake_case`.
 
+## Credentials
+
+Credentials can be initiated as described in the [getting started page](https://sentry-docs-git-relayconfig.sentry.dev/product/relay/getting-started/?original_referrer=https%3A%2F%2Fgithub.com%2Fgetsentry%2Fsentry-docs%2Fpull%2F7062#creating-credentials), but can be overriden by setting the following environment variables:
+
+secret key: `"RELAY_SECRET_KEY"`
+public key: `"RELAY_PUBLIC_KEY"`
+id: `"RELAY_ID"`
```

Comment:
This renders in a row. What do you think about creating an actual list for this?

Reply:
I think a list will be easier to quickly read :)

Reply:
i made it now an unordered list